client/disputes/utils.ts

```diff
@@ -67,14 +67,14 @@ export const isUnderReview = ( status: DisputeStatus | string ): boolean => {
 	return disputeUnderReviewStatuses.includes( status );
 };
 
-export const isInquiry = ( dispute: Pick< Dispute, 'status' > ): boolean => {
+export const isInquiry = ( status: DisputeStatus ): boolean => {
 	// Inquiry dispute statuses are one of `warning_needs_response`, `warning_under_review` or `warning_closed`.
-	return dispute.status.startsWith( 'warning' );
+	return status.startsWith( 'warning' );
```

Comment:
Do we have a test for this change already? If not, do we plan to write it with this change? 😄

Reply:
What bugs or regressions would that test catch? How valuable would that test be?

Reply:
I did not think of adding a test, earlier, since the function has a very definite set of input - Dispute statuses, and a single line logic. Hence, very low chance of any regression unless some changes are made to the logic itself, or if the set of statuses change. I kept an eye though, on any failing tests of functions calling this function - `isInquiry` 

I have added a test in this commit - https://github.com/Automattic/woocommerce-payments/pull/8312/commits/ad6b7656f90789cb911c739bbbf099613922e8d6 , to make sure the function returns its expected values.

Reply:
>What bugs or regressions would that test catch? How valuable would that test be?

As @nagpai added, I think covering the behavior of what is expected from that function in isolation. It can regress in case of some changes made by mistake.